src/Azure.Functions.Cli/Common/ProgrammingModel.cs

```diff
@@ -0,0 +1,8 @@
+namespace Azure.Functions.Cli.Common
+{
+    public enum ProgrammingModel
+    {
+        Default,
+        Preview
```

Comment:
I agree v1/v2 is a good option here and we should use those values to ensure clarity when the default changes. It will also align somewhat with the conversation we were having around properties in metadata.json for templates.

Reply:
I agree, I think v1/v2 will be a direct way to reference the programming models in the documentation, and ultimately provide an intuitive user experience that differentiates between the two models. @michaelpeng36, is it possible to replace Default and Preview with v1 and v2?

Reply:
Done.